specs/theme/highlight.js

```diff
@@ -0,0 +1,1328 @@
+/*
+  Highlight.js 10.7.2 (00233d63)
```

Comment:
[no change needed] this file looks :suspect: but it seems intentional
1. https://rust-lang.github.io/mdBook/format/theme/syntax-highlighting.html
2. https://highlightjs.org/
3. https://github.com/celestiaorg/celestia-specs/blob/master/theme/highlight.js

Reply:
It's copy-pasted from the original repo without change.